src/features/home/ui/ChartHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import {
+  ChartHeaderContainer,
+  DateContainer,
+  HeaderDescription,
+  HeaderTitle,
+  IcArrow,
+  YearMonth,
+} from './chartHeader.style';
+import { IcArrowNext, IcArrowPrev } from '@/shared/assets/svg/index';
```

Comment:
```suggestion
import { IcArrowNext, IcArrowPrev } from '@/shared/assets/svg';
```
로 선언해도 인식이 될 거 같습니다!

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/89/commits/7367c94da447393642e1f70b6376f457fc1f7977 에서 변경했습니다!!